app/graphql/types/customers/object.rb

```diff
@@ -28,11 +27,11 @@ class Object < Types::BaseObject
       field :payment_provider, Types::PaymentProviders::ProviderTypeEnum, null: true
       field :payment_provider_code, String, null: true
       field :phone, String, null: true
-      field :state, String, null: true
       field :tax_identification_number, String, null: true
       field :timezone, Types::TimezoneEnum, null: true
       field :url, String, null: true
-      field :zipcode, String, null: true
+
+      field :shipping_address, Types::Customers::ShippingAddress, null: true
```

Comment:
In my mind it was something like:

```suggestion
      field :shipping_address, Types::Customers::Address, null: true
```

But not sure it could be used easily for the base address